zbus/src/connection/mod.rs

```diff
@@ -1541,14 +1541,14 @@ mod tests {
 #[cfg(feature = "p2p")]
 #[cfg(test)]
 mod p2p_tests {
-    use futures_util::stream::TryStreamExt;
+    use event_listener::Event;
+    use futures_util::{StreamExt, TryStreamExt};
     use ntest::timeout;
     use test_log::test;
     use zvariant::{Endian, NATIVE_ENDIAN};
 
-    use crate::{conn::AuthMechanism, Guid};
-
-    use super::*;
+    use super::{socket, Builder, Connection};
+    use crate::{conn::AuthMechanism, Guid, Message, MessageStream, Result};
```

Comment:
Expanding the `super::*` import was required to prevent `futures_lite::StreamExt` from being imported here, which has some method clashes with one or both of the `futures_util` extension traits.

Reply:
No worries. I'm not a big fan of wildcard imports anyway and importing from `super` is where I sometimes make/allow an exception. :)